### ISSUE TITLE
[Bug] Handhelds will Start to Drift Away in the Codex

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#5327]
### Short Description of the issue:
Handhelds held by the mouse in the Codex start to drift away. It snaps back when you place the Handheld back on your hotbar so no harm is done, it just seems to be a visual bug

https://github.com/tsunamayo/Starship-EVO/assets/27857711/b7efbf0e-d71e-47ba-abff-072c597cd4ed

### Steps to reproduce the issue:
- Place a Handheld on your hotbar (does not need to be the current active slot)
- Open the Codex and Grab it with your mouse
- The handheld will start to drift forward away from the character

### Environment
- Starship EVO version: 23w41a
- Graphics Card: RX 6700XT

**Comment by tsunamayo** [comment]
@ProPeach yes I think it was already reported. A bit tricky to fix without introducing other issue so it is low on the list unfortunately. thx
